test/selenium/README.md

```diff
@@ -22,18 +22,106 @@ Run the [run-tests script](https://github.com/cockpit-project/cockpit/blob/maste
+  - Pass ``--sit`` parameter to ``run-tests`` which will leave all test machines running after the tests finish.
+  - Use own selenium grid via option ``--hub``
+
+#### Use selenium grid in container
+Start own selenium grid in container ([Original description](https://github.com/SeleniumHQ/docker-selenium#debugging)).
+
+  - Create selenium grid with chrome browser with VNC enabled (you can replace it by firefox and change commands according to that):
+    ```
+    podman run --rm -p 4444:4444 -p 5555:5900 -v /dev/shm:/dev/shm selenium/standalone-chrome-debug
+    ```
+  - Start VNC session to that machine (password is ``secret``):
+    ```
+    vncviewer 127.0.0.1:5555
+    ```
+  - Enable port forwarding of port ``127.0.0.2:9091`` to publicly accessible address, because selenium hub in podman container will have trouble to contact host port, because it is forwarded right to ``127.0.0.2``. ``YOUR_IP`` is host name or ip adress of your machine, that selenium ``HUB`` will be able to access.
+    ```
+    ssh -L $YOUR_IP:9991:127.0.0.2:9091 $(whoami)@localhost
+    ```
+  - Run the test. ``HUB_IP`` is ip address of your selenium grid, has to be able to see cockpit instance via port 9991 on ``GUEST_IP``
+    ```
+    test/selenium/run-tests --hub $HUB_IP:$GUEST_IP:9991 -v -b chrome selenium-base.py --sit
+    ```
+
+#### Selenium grid on your machine with your local browsers
+Steps are described in [selenium page](https://www.browserstack.com/guide/selenium-grid-tutorial)
+  - Download selenium standalone:
+    ```
+    curl -f -L https://selenium-release.storage.googleapis.com/3.141/selenium-server-standalone-3.141.59.jar > selenium.jar
+    ```
+  - Run hub and attach browsers here:
+    ```
+    java -jar selenium.jar -role hub
+    ```
+    - Then attach your local browsers there. You have browsers eg. firefox or chrome installed, and also ensure you have proper [selenium drivers for selected the browser](https://github.com/cockpit-project/cockpit/tree/master/test/selenium#which-browser-to-use)
+      ```
+      java -jar selenium.jar -role node -hub https://localhost:4444/grid/register
+      ```
+  - Run the test. ``HUB_IP`` is ip address of your selenium grid
+    ```
+    test/selenium/run-tests --hub $HUB_IP -v -b chrome selenium-base.py --sit
+    ```
+#### Reschedule the the test/s again
+It is possible to reschedule tests if you used ``--sit`` option for ``run-tests``, in case of failure, machines won't be destroyed.
+Command-line options depend on which selenium grid you used.
+It is described on the output of the ``run-tests`` script. Eg.:
+
+```
+$test/selenium/run-tests --hub $HUB_IP -v -b chrome selenium-navigate.py --sit
+
+...
+To rerun tests connect to VM machine with avocado installed via
```

Comment:
Here you have some text inside '`', please leave only the commands inside the quotes